code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -11,6 +11,7 @@
 	else
 		new /obj/item/storage/backpack/satchel/cap(src)
 	new /obj/item/clothing/neck/cloak/cap(src)
+	new /obj/item/clothing/neck/petcollar(src)

```

Comment:
captain doesn't have a pet on all maps

Reply:
Renault you literal rarely plays.

Reply:
not on all maps you literal furry

Reply:
Well then maybe you should get better grammar and suck it up, you literal deuce.

Reply:
renault got added to box to and the most main maps is box/meta

Reply:
should put it in the locker on the maps that have it instead